packages/shared/wallet/src/settings-side-drawer/settings-side-drawer.tsx

```diff
@@ -190,11 +191,10 @@ const WalletButton = styled('div', {
   `
 );
 
-const Title = styled(Typography)(
+const Title = styled(LabelValue)(
```

Comment:
Maybe we can also make this "SideNavText" as well

Reply:
SideNavText is not the correct font size according to the mocks

Reply:
Which is why I went with LabelValue